### PR TITLE
IGNITE-14595 Implement ExpiryPolicy support.

### PR DESCRIPTION
**Pull request by ivandasch**

### CODE REVIEW

**Review by isapego — COMMENTED** [review]
Please see my comments inline.

File: examples/expiry_policy.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+from pyignite.datatypes import ExpiryPolicy
+from pyignite.datatypes.prop_codes import PROP_NAME, PROP_EXPIRY_POLICY
+
+
+def main():
+    print("Running sync ExpiryPolicy example.")
+
+    client = Client()
+    with client.connect('127.0.0.1', 10800):
+        if not client.protocol_context.is_expiry_policy_supported():
```
[the remark this answers is not in the document]
Comment (ivandasch) [reply examples/expiry_policy.py]: Yep, changed to catch Exception

File: examples/expiry_policy.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+            print("'ExpiryPolicy' is not supported by cluster.")
```
[the remark this answers is not in the document]
Comment (ivandasch) [reply examples/expiry_policy.py]: Yep, fixed

File: examples/failover.py [hunk 3]
```diff
@@ -26,31 +26,30 @@
 ]
 
 
-def main():
```
[the remark this answers is not in the document]
Comment (ivandasch) [reply examples/failover.py:29]: This was accidential mistake from previous commits, I just fixed it.

**Pull request closed without merging**